### PR TITLE
LPS-155561 Validate if there is at least one custom object field

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-155561

### CODE REVIEW

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java [hunk 1]
```diff
@@ -486,10 +486,9 @@ public ObjectDefinition publishCustomObjectDefinition(
 			throw new ObjectDefinitionStatusException();
 		}
 
-		int count = _objectFieldPersistence.countByObjectDefinitionId(
-			objectDefinition.getObjectDefinitionId());
+		if (_objectFieldPersistence.countByODI_NotS(objectDefinitionId, true) ==
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java:489]: You can combine this PR with this one that I just reviewed it: https://github.com/liferay-objects/liferay-portal/pull/687

**Comment by mateussgn** [comment]
ci:close

**Comment by mateussgn** [comment]
Resent on https://github.com/liferay-objects/liferay-portal/pull/698

**Pull request closed without merging**